.github/workflows/docker-action.yml

```diff
@@ -16,17 +16,19 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        coq_version:
-          - 'dev'
-          - '8.19'
-          - '8.18'
+        image:
+          - 'coqorg/coq:8.20'
+          - 'coqorg/coq:8.19'
+          - 'coqorg/coq:8.18'
+          - 'rocq/rocq-prover:dev'
+          - 'rocq/rocq-prover:9.0'
```

Comment:
is using `image` more recommended than coq_version?

Reply:
No, it's not specifically recommended when manually writing a GitHub workflow file. But this one is auto-generated from the Rocq-community templates, which are programmed in a poorly expressive templating language (Mustache), and we want them to support the option of specifying an image (e.g., for MathComp-based projects).